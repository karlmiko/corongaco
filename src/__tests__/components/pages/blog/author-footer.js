import React from 'react'
import renderer from 'react-test-renderer'
import AuthorFooter from '~components/pages/blog/author-footer'

describe('Components : Pages : Blog : Author Footer', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <AuthorFooter
          authors={[
            {
              name: 'Name Name',
              twitterLink: 'https://twitter.com/namename',
              twitterHandle: 'namename',
              link: 'https://www.website.com',
              childContentfulAuthorBiographyTextNode: {
                childMarkdownRemark: {
                  html:
                    '<p>Name Name is a an author at <em>A Publication</em>.</p>',
                },
              },
              headshot: {
                file: {
                  fileName: 'filename.png',
                },
                resize: {
                  width: 200,
                  height: 200,
                  src: '//images.ctfassets.net/file',
                },
              },
            },
          ]}
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
