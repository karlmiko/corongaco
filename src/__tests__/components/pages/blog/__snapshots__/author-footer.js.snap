// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Blog : Author Footer renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="authorFooter"
  >
    <img
      alt="filename.png"
      src="//images.ctfassets.net/file"
    />
    <div
      className="textContent"
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "<p>Name Name is a an author at <em>A Publication</em>.</p>",
          }
        }
      />
      <a
        href="https://twitter.com/namename"
      >
        @
        namename
      </a>
    </div>
  </div>
</div>
`;
